src/main/java/com/bravo/user/validator/UserValidator.java

```diff
@@ -14,6 +17,20 @@ public void validateId(String id){
       throw new BadRequestException("'id' is required");
     }
   }
+  
+  /**
+   * Validates a user name to ensure it is not null, not empty, and does not contain space, numbers, or special characters.
+   * 
+   * @param name User name to validate
+   * @throws BadRequestException if the name is invalid
+   */
+  public void validateName(String name) {
+	  Pattern pattern = Pattern.compile("[^a-z*%!]", Pattern.CASE_INSENSITIVE);
```

Comment:
This `Pattern` could be `private static final`

Reply:
Good call.  Will update.